manifests/client.pp

```diff
@@ -7,6 +7,10 @@
 # @param manage_repository
 #   Manage the repository for package installation
 #
+# @param include_clientlaunch_d
+#   Ensure, that clientlaunch.d/*.cfg is included in clientlaunch.cfg - some xymon packages miss to do so.
+#   Will not remove any include value false!
```

Comment:
Could you state this more precisely? What value isn't removed? From where? What is false?
I don't get it.

Reply:
I will to describe it more precisely

Reply:
Hope it is more understandable now?